src/main/java/co/cask/hydrator/plugin/source/KafkaConfig.java

```diff
@@ -155,24 +158,21 @@ public String getBrokers() {
     return brokers;
   }
 
-  @Nullable
   public String getTimeField() {
-    return Strings.isNullOrEmpty(timeField) ? null : timeField;
+    return Strings.isNullOrEmpty(timeField) ? "timestamp" : timeField;
   }
 
-  @Nullable
   public String getKeyField() {
-    return Strings.isNullOrEmpty(keyField) ? null : keyField;
+    return Strings.isNullOrEmpty(keyField) ? "key" : keyField;
```

Comment:
You can set up the default value in the constructor like https://github.com/hydrator/kafka-plugins/blob/d50b4e60b2995f64b542beef514d66c5924cd5de/src/main/java/co/cask/hydrator/plugin/source/KafkaConfig.java#L126 same as `KafkaBatchSource`. Otherwise it will be confused when we directly use config.keyField, as this will give null value.